schemas/com.vimeo/chapter_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a Vimeo Chapter.",
+  "self": {
+    "vendor": "com.vimeo",
+    "name": "chapter_event",
```

Comment:
I could only find a chapter change event in the docs – is it this one? If so, maybe `chapter_change_event` would be a more descriptive name?

Reply:
Makes sense, renamed.